components/pluralrules/src/bin/generate_res.rs

```diff
@@ -0,0 +1,19 @@
+use icu_pluralrules::data::io::json::get_resource;
+use icu_pluralrules::PluralRuleType;
+use std::fs::File;
+use std::io::prelude::*;
+
```

Comment:
A short blurb explaining what the binary is supposed to do, would be most welcome.